core/app/models/spree/order.rb

```diff
@@ -54,6 +54,9 @@ class CannotRebuildShipments < StandardError; end
     deprecate temporary_credit_card: :temporary_payment_source, deprecator: Spree::Deprecation
     deprecate :temporary_credit_card= => :temporary_payment_source=, deprecator: Spree::Deprecation
 
+    has_one :from_merged_order, foreign_key: :merged_to_order_id, class_name: 'Spree::Order', inverse_of: :merged_to_order
```

Comment:
I recall that a single orders can be the result of merging multiple orders. Should this be a `has_many`?

https://github.com/solidusio/solidus/blob/9f99aa6f084ea3c9d1015edc6c39fb119b169b8d/core/lib/spree/core/controller_helpers/order.rb#L60-L66

Reply:
@elia thank you for pointing this out, I did not realize this could happen as well.